README.md

```diff
@@ -170,5 +241,8 @@ Write what an example ERB file (aka view) might look like to list all the warrio
 
 Write your code here (**NOTE: syntax highlighting doesn't work for ERB in markdown files, so ignore the colors!**):
 ```html
-<!-- code here -->
+<% @warriors.each do |warrior| %>
+  <p><%= warrior %></p>
+<% end %>
+
```

Comment:
really close here all you need to add is `.name` should look like this. `<p><%= warrior.name %></p>`